File: transpiler/antlr/paspy/src/main/examples/ifprog.py
def ifprog():
	number = 50
	print('If this was a grade, you would receive a grade of:  ')
	if number >= 90:
		print('A')
	elif number >= 80:
		print('B')
	elif number >= 70:
		print('C')
	elif number >= 60:
		print('D')
	else:
		print('F')

	print()


if __name__ == "__main__":
	ifprog()

File: transpiler/antlr/paspy/src/main/examples/ifprog.pas
program ifprog;
 
(*** Demonstrate the if.  ***)
 
var
   number: integer;
 
(* **************************    main      ********************************** *)
 
begin
  number := 50;
 
  write('If this was a grade, you would receive a grade of:  ');
  if number >= 90 then
     write('A')
  else if number >= 80 then
     write('B')
  else if number >= 70 then
     write('C')
  else if number >= 60 then
     write('D')
  else
     write('F');
  writeln;
end.